build.gradle.kts

```diff
@@ -95,9 +95,18 @@ dependencies {
 
     testCompile("junit:junit:4.12")
     testCompile("org.assertj:assertj-core:3.11.0")
-    testCompile("com.atlassian.performance.tools:docker-infrastructure:0.1.2")
-    testCompile("com.github.docker-java:docker-java:3.1.1")
-    testCompile("com.atlassian.performance.tools:infrastructure:[4.0.0,5.0.0)")
+    testCompile("com.atlassian.performance.tools:docker-infrastructure:0.3.3")
+    /*
+     * Transitively pulls in their shaded `DockerClientImpl`.
+     * Gotta hack it this way until any of the following happens:
+     * - they stop shading docker-java-core
+     * - they shade docker-java-core properly
+     * - we stop using testcontainers in docker-infrastructure
+     * - we stop using docker-infrastructure
+     */
+    testCompile("org.testcontainers:testcontainers:1.14.3")
+    testCompile("com.github.docker-java:docker-java-api:3.2.5")
+    testCompile("com.atlassian.performance.tools:infrastructure:[4.0.0,4.15.0]")
```

Comment:
Note the message in 53acb3781ecb1cd5c37ddb53393ecbcbc55167cc.
Here's an upside of independently released modules. We can bump everything and roll back just the one problematic module to a working version.

Reply:
Ok, I narrowed it down. Using `4.15.1` brings the `Error response from daemon: docker-containerd-shim not installed on system` error. So 4.15.0 is indeed the highest working version.